File: paoc/paoc.view.web/src/main/java/br/cefetmg/paoc/negocio/dto/Log.java
package br.cefetmg.paoc.negocio.dto;

public class Log {
    private String acao;
    private String autor;
    private String dataHora;
    private int cod;
    private String objeto;

    public Log(String acao, String autor, String dataHora, int cod, String objeto) {
        this.acao = acao;
        this.autor = autor;
        this.dataHora = dataHora;
        this.cod = cod;
        this.objeto = objeto;
    }

    public String getAcao() {
        return acao;
    }

    public String getAutor() {
        return autor;
    }

    public String getDataHora() {
        return dataHora;
    }

    public int getCod() {
        return cod;
    }

    public String getObjeto() {
        return objeto;
    }
    
    
}

File: services/Log.php
<?php 
class Log {
    private $acao;
    private $autor;
    private $dataHora;
    private $cod;
    private $objeto;

    public function __construct($acao, $autor, $dataHora, $cod, $objeto) {
        $this->acao = $acao;
        $this->autor = $autor;
        $this->dataHora = $dataHora;
        $this->cod = $cod;
        $this->objeto = $objeto;
    }

    public function getAcao() {
        return $this->acao;
    }

    public function getAutor() {
        return $this->autor;
    }

    public function getDataHora() {
        return $this->dataHora;
    }

    public function getCod() {
        return $this->cod;
    }

    public function getObjeto() {
        return $this->objeto;
    }
}

?>